src/iocore/cache/Cache.cc

```diff
@@ -645,45 +634,22 @@ CacheProcessor::cacheInitialized()
           break;
         }
       }
+
+      int64_t http_ram_cache_size = 0;
+
       // let us calculate the Size
       if (cache_config_ram_cache_size == AUTO_SIZE_RAM_CACHE) {
         Dbg(dbg_ctl_cache_init, "CacheProcessor::cacheInitialized - cache_config_ram_cache_size == AUTO_SIZE_RAM_CACHE");
-        for (i = 0; i < gnvol; i++) {
-          stripe = gvol[i];
-
-          if (gvol[i]->cache_vol->ramcache_enabled) {
-            gvol[i]->ram_cache->init(stripe->dirlen() * DEFAULT_RAM_CACHE_MULTIPLIER, stripe);
-            ram_cache_bytes += gvol[i]->dirlen();
```

Comment:
This is another bug, this should be multiplied by `DEFAULT_RAM_CACHE_MULTIPLIER`.